contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,69 @@
+first step in making sure you are helping the docs team create content that is fully modularized.
+
+== What is the branch setup for modularized docs?
+
+* link:https://github.com/openshift/openshift-docs/tree/enterprise-4.0[enterprise-4.0]
++
+This builds both OCP 4.0 and Origin/Upstream content. Origin content will be separated using
+ifdef statements in content and distro statements in the _topic_map.yml.
++
+Only CCS submits and merges their PRs here till OCP 4.0 is released, unless others are keen to submit
+content in modularized format.
+
+* https://github.com/openshift/openshift-docs/tree/master[master]
+This builds legacy (existing), mostly non user story based content. This branch will continue to update
+docs.okd.io (upstream) till we switch to 4.0 with the enterprise-4.0 branch.
++
+Please continue to submit your PRs to this branch and specify if it is for 3.11 and/or 4.0.
+If the submitted PR is for the 3.11 branch, CCS merges with follow up edits in 3.11.
+CCS then creates user stories for its inclusion in enterprise-4.0 branch, and if it is completely new content,
+in the enterprise-3.11 branch as well.
+
+* https://github.com/openshift/openshift-docs/tree/enterprise-3.11[enterprise-3.11]
+This branch contains content for the 3.11 branch, and the last branch that contains content that is not completely
+modularized or written from scratch.
++
+This may contain user stories for content for completely new features.
+
+== Do you have a template for submitting user stories if I don't want o contribute content in a modularized format?
+Yes, and this assumes that you are supplying the user story for CCS to create the modularized content from.
+
+Use the following template to provide context for your user story.
+
+=== Feature info
+* What is the feature being developed? What does it do?
+* How does it work?
+* Are there any configuration files/settings/parameters being added or modified? Are any new commands being added or modified?
+* What tools or software does the docs team need to test how this feature works? Does the docs team need to update any installed software?
+* Are there any existing blogs, Wiki posts, Kbase articles, or Bzs involving this feature? Or any other existing information that may help to understand this feature?
+
+=== Product info
+* Is this feature being developed for Online? Enterprise? Dedicated? Origin? All?
+* Will this feature be rolled back to previous versions?
+* If it's for Online, what type of plan do users need to use this feature?
+* Is it user-facing, or more behind-the-scenes admin stuff?
+* What tools or software does the docs team need to test how this feature works?
+
+=== Customer impact
+* Who is the intended audience for this feature? If it's for Enterprise, does it apply to developers, admins, or both?
+* Why is it important for our users? Why would they want to use this feature? How does it benefit them?
```

Comment:
Please swap lines 66 and 67.

Reply:
I think these can stay in that order. 
first who is the user
why is it important to that user.
We need to know the user before we can know the why ??